tests/smoke/test_notebooks_python.py

```diff
@@ -91,3 +91,26 @@ def test_vw_deep_dive_smoke(notebooks):
     assert results["ndcg"] == pytest.approx(0.65339, rel=TOL, abs=ABS_TOL)
     assert results["precision"] == pytest.approx(0.514738, rel=TOL, abs=ABS_TOL)
     assert results["recall"] == pytest.approx(0.25684, rel=TOL, abs=ABS_TOL)
+
```

Comment:
hey @motefly, do you know how long does this take?

Reply:
For the lightgbm_quickstart, it takes about 60 seconds.

Reply:
awesome, thanks